src/main/java/com/dynatrace/loadrunner/CommandLineParser.java

```diff
@@ -0,0 +1,56 @@
+package com.dynatrace.loadrunner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class CommandLineParser {
+
+	final public Map<String, String> arguments;
+
+	public CommandLineParser(String[] args) {
+
+		arguments = new HashMap<>();
+
+		for (int i = 0; i < args.length; i++) {
+			String key = null;
+			if (args[i].startsWith("-")) {
+				key = args[i].substring(1);
+				if ((i + 1) < args.length) {
+					if (args[i + 1].charAt(0) != '-') {
+						arguments.put(key, args[i + 1]);
+						i++;
+					} else {
+						arguments.put(args[i], null);
+					}
+				} else {
+					arguments.put(args[i], null);
+				}
+			} else {
+				/*
+				 * If param doesn't start with '-', then we assume key doesn't have a value thus
```

Comment:
but we do the same above for all the other arguments also (those starting with '-')